src/fabric/RNGestureHandlerRootView.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { PropsWithChildren } from 'react';
+import { ViewProps } from 'react-native';
+
+// @ts-ignore react-native-codegen does not support TypeScript yet
+import GestureHandlerRootViewNativeComponent from './RNGestureHandlerRootViewNativeComponent';
+
+export interface GestureHandlerRootViewProps
+  extends PropsWithChildren<ViewProps> {}
+
+export default function GestureHandlerRootView({
+  children,
+  ...rest
+}: GestureHandlerRootViewProps) {
+  return (
+    <GestureHandlerRootViewNativeComponent {...rest}>
+      {children}
+    </GestureHandlerRootViewNativeComponent>
+  );
+}
```

Comment:
I think that `<GestureHandlerRootViewNativeComponent {...rest}>{childres}</...>` is the same as `<GestureHandlerRootViewNativeComponent {...props} />`. 

Is there a specific reason why you divide the props here?

Reply:
It seems like the `fabric/...` wrappers weren't necessary at all. But your comment also applies to the older `GestureHandlerRootView.android.tsx`. I don't think there is a reason for it to be this way. I changed it to match `GestureHandlerRootView.tsx` in https://github.com/software-mansion/react-native-gesture-handler/pull/1893/commits/50f054f59637198cecad15875547887d0a6ac2e4.